kerneltest/app.py

```diff
@@ -171,7 +171,7 @@ def is_admin(user):
         return False
 
     admins = APP.config['ADMIN_GROUP']
-    if isinstance(admins, basestring):
+    if isinstance(admins, str):
```

Comment:
This will break the behavior for py2, so :-1: as is.

Using six we may be able to be both py2 and py3 compatible.